assignments/callbacks.js

```diff
@@ -41,29 +41,57 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+    cb(arr[0]);
```

Comment:
This is not passing the length of the array into the callback. This is passing the first item in the array.